lib/LaTeXML/Package/OmniBus.cls.ltxml

```diff
@@ -59,8 +59,12 @@ DefEnvironment('{frontmatter}', '#body');
 DefEnvironment('{mainmatter}',  '#body');
 DefEnvironment('{backmatter}',  '#body');
 
-DefMacro('\shorttitle{}',  '\@add@frontmatter{ltx:toctitle}{#1}');
-DefMacro('\subtitle{}',    '\@add@frontmatter{ltx:subtitle}{#1}');
+DefMacro('\shorttitle{}', '\@add@frontmatter{ltx:toctitle}{#1}');
+DefMacro('\subtitle{}',   '\@add@frontmatter{ltx:subtitle}{#1}');
+DefMacro('\title[]{}',
+  '\if.#1.\else\def\shorttitle{#1}\@add@frontmatter{ltx:toctitle}{#1}\fi'
+    . '\def\@title{#2}\@add@frontmatter{ltx:title}{#2}', locked => 1);
+
```

Comment:
Shouldn't we just end up with LaTeX.pool's (newly enhanced) `\title`?

Reply:
Agreed, that suffices. I've committed the change.